test_requirements.txt

```diff
@@ -1,11 +0,0 @@
-certifi==2018.8.24
```

Comment:
If we remove this file we need to update the `README.md` for how to install test requirements.